homeassistant/components/life360/device_tracker.py

```diff
@@ -0,0 +1,617 @@
+"""Support for Life360 device tracking."""
+
+from collections import namedtuple
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.device_tracker import (
+    CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL,
+    ENTITY_ID_FORMAT as DT_ENTITY_ID_FORMAT, PLATFORM_SCHEMA)
+from homeassistant.components.zone import (
+    create_zone, ENTITY_ID_HOME, ICON_HOME, remove_zone)
+from homeassistant.components.zone.zone import active_zone
+from homeassistant.const import (
+    ATTR_BATTERY_CHARGING, ATTR_ENTITY_ID, ATTR_FRIENDLY_NAME, ATTR_LATITUDE,
+    ATTR_LONGITUDE, ATTR_NAME, CONF_EXCLUDE, CONF_FILENAME, CONF_INCLUDE,
+    CONF_PASSWORD, CONF_PREFIX, CONF_USERNAME, LENGTH_FEET, LENGTH_KILOMETERS,
+    LENGTH_METERS, LENGTH_MILES, STATE_HOME, STATE_UNKNOWN)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.event import track_time_interval
+from homeassistant.util.distance import convert
+import homeassistant.util.dt as dt_util
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_FILENAME = 'life360.conf'
```

Comment:
Use storage helper instead of creating your own files.

Reply:
The file management is done in the pypi package which doesn't know about HA's storage helpers, similar to the way googe_maps does it. But since I own the pypi package, I suppose I can find a way to make this happen.

Reply:
In that case, prefix the file with a `.` so it's hidden. However, by having your own file, it will not be part of our future backup/restore system.